EZSource/Classes/Extensions/UITableViewCellExtension.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  UITableViewCellExtension.swift
+//  EZSource
+//
+//  Created by Alex Hmelevski on 2018-06-04.
+//
+
+import Foundation
+
+extension UITableViewCell: TableViewCell {
+    public var uiTableViewCell: UITableViewCell { return self}
```

Comment:
It should be like this: `public var uiTableViewCell: TableViewCell { return self}` no?

Reply:
No that's the thing TableViewCell is a protocol... we need UITableViewCell to be able to erase the type an be able to use in  `cellForRowAtIndexPath`